app/templates/ui/app/user/user.service.spec.js

```diff
@@ -49,46 +53,61 @@
 
     it('should get the current logged in user - if loginService not init', function () {
       service.getUser().then(function(user) {
-        expect(user).to.deep.eq({ name: 'bob' });
+        expect(user).to.deep.eq(null);
       });
-      expect($http.get).to.have.been.calledOnce;
-      $rootScope.$apply();
-    });
 
-    it('should set user with valid credentials', function () {
-      loginService.login('test', 'abc').then(function(response) {
-        expect(response.data).to.deep.eq({ authenticated: true, username: 'bob' });
-        expect(service.currentUser()).to.deep.eq({ name: 'bob' });
-        expect(service.getUser()).to.deep.eq({ name: 'bob' });
-      });
-      expect($http.post).to.have.been.calledOnce;
-      $rootScope.$apply();
-    });
+      expect(loginService.getAuthenticatedStatus).to.have.been.calledOnce;
 
-    it('should not set user with invalid credentials', function () {
-      _user.data.authenticated = false;
-      loginService.login('test', 'abcd').then(function(response) {
-        expect(response.data.authenticated).to.be.false;
-        expect(service.currentUser()).to.not.be.defined;
-      });
-      expect($http.post).to.have.been.calledOnce;
       $rootScope.$apply();
     });
 
-    it('should clear user after logout', function () {
-      expect(service.currentUser()).to.not.be.defined;
-      _user.data.authenticated = true;
-      loginService.login('test', 'abcd').then(function(response) {
-        expect(response.data).to.deep.eq({ authenticated: true, username: 'bob' });
-        expect(service.currentUser()).to.deep.eq({ name: 'bob' });
-
-        loginService.logout().then(function(response) {
-          expect(loginService.isAuthenticated()).to.be.false;
-          expect(service.currentUser()).to.not.be.defined;
-        });
-      });
+    // it('should login correctly using loginService', function () {
+    //   $rootScope.$broadcast('loginService:login-success', _user);
+    //   $rootScope.$digest(service);
 
-      $rootScope.$apply();
-    });
```

Comment:
Hmm, odd. user and login stuff didn't change much lately. Must be something small that causes it to fail..